packages/manager/modules/nutanix/src/dashboard/general-info/redeploy/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,58 @@
+{
+  "nutanix_redeploy_title": "Redéploiement du cluster",
+  "nutanix_redeploy_warning_info_1": "Vous conservez vos nœuds physiques et votre facturation reste inchangée.",
+  "nutanix_redeploy_warning_info_2": "Veuillez cependant noter les informations suivantes :",
+  "nutanix_redeploy_warning_info_3": "toutes les données présentes sur votre cluster seront effacées ;",
+  "nutanix_redeploy_warning_info_4": "vos codes d’accès Prism seront regénérés ;",
+  "nutanix_redeploy_warning_info_5": "vos configurations dans Prism seront effacées ;",
+  "nutanix_redeploy_warning_info_6": "la configuration réseau sera réinitialisée aux valeurs par défaut (IP, vRack, etc.).",
+  "nutanix_redeploy_step1_title": "Méthode de redéploiement du cluster Nutanix",
+  "nutanix_redeploy_step2_title_initial_configuration": "Récapitulatif elements techniques",
+  "nutanix_redeploy_step2_title_custom_configuration": "Stratégie de déploiement & résilience",
+  "nutanix_redeploy_initial_configuration": "Redéployer la configuration initiale à l’identique",
+  "nutanix_redeploy_initial_configuration_prism_central": "Prism Central:",
+  "nutanix_redeploy_initial_configuration_prism_central_description": "Prism Central est l’interface vous permettant d’administrer différents clusters Nutanix. Et ce, bien qu’ils soient séparés physiquement. ",
+  "nutanix_redeploy_initial_configuration_prism_central_type": "Type:",
+  "nutanix_redeploy_initial_configuration_prism_central_type_description": "Le champ « Type » détermine le niveau de disponibilité de la machine virtuelle (VM) Prism Central que vous souhaitez appliquer. Chaque VM consomme une partie des ressources de votre cluster. Le type « Alone » signifie qu'une seule VM est déployée. Le type « Scale-out » signifie que trois machines sont déployées sur votre cluster pour gérer Prism Central. Cela affecte la capacité d'évolution des VM ainsi que la résilience des systèmes.",
+  "nutanix_redeploy_initial_configuration_prism_central_virtual_ip": "IP virtuelle (VIP):",
+  "nutanix_redeploy_initial_configuration_prism_central_virtual_ip_description": "VIP si l'IP flottante est utilisée pour exposer Prism Central. Cette IP sera configurée dans le back-end des IP publiques Load Balancer de votre cluster.",
+  "nutanix_redeploy_initial_configuration_prism_central_ip": "IP:",
+  "nutanix_redeploy_initial_configuration_prism_central_ip_description": "IP privées des VM Prism Central.",
+  "nutanix_redeploy_initial_configuration_replication": "Redundancy Factor:",
```

Comment:
**question:** Should it be translated? If it's not supposed to be translated, then it's a constant.